.github/workflows/neo4j-migrations.yml

```
@@ -0,0 +1,63 @@
+          cd neo4j/tools
+          npm install
+      
+      - name: Run and verify migrations
+        run: |
+          cd neo4j/tools
+          ./verify_migration.sh
+        env:
+          NEO4J_URI: bolt://localhost:7687
+          NEO4J_USER: neo4j
+          NEO4J_PASSWORD: password
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)